src/fragments/start/getting-started/angular/hosting.mdx

```diff
@@ -0,0 +1,52 @@
+You've successfully built your first app with Amplify! Now that you've built something, it's time to deploy it to the web with Amplify Hosting!
+
+## Add hosting to your app
+You can manually deploy your web app or setup automatic continuous deployment. In this guide we'll cover how to manually deploy and host your static web app to quickly share with others. If you want to learn about continuous deployment instead, please follow [this guide](https://docs.aws.amazon.com/amplify/latest/userguide/multi-environments.html#standard).
+
+From the root of your project, run the following command and select the **bolded options**. 
+```bash
+amplify add hosting
+```
+
+```console
+? Select the plugin module to execute: # Hosting with Amplify Console (Managed hosting with custom domains, Continuous deployment)
+? Choose a type: # Manual Deployment
+```
+
+## Publish your app
+
+Run the following command to publish your app.
+
+```bash
+amplify publish
+```
+
+<Callout>
+  If you get "Error: bundle initial exceeded maximum budget" error based on bundle size, you need to bump the maximum bundle size budget for your Angular application. Find the `budgets` keyword in `angular.json`, and bump `maximumError`:
+
+```json
+{
+  // ...
+  "budgets": [
+    {
+      "type": "initial",
+      "maximumWarning": "500kb",
+      "maximumError": "2mb"
+    },
+    // ...
+  ]
+}
+```
+</Callout>
+
+Congratulations, your app is online!
+
+![image](/images/browser-published-updated.png))
```

Comment:
I think there is an extra `)` on this line and so it shows up in the page as a stray character

Reply:
TY!
```suggestion
![image](/images/browser-published-updated.png)
```